packages/web/webpack.config.js

```diff
@@ -53,8 +53,15 @@ module.exports = (env, argv) => {
             hot: false,
             compress: true,
             historyApiFallback: true,
-        } 
-        : {},
```

Comment:
why is this change necessary?

Reply:
I belive that was just a little format fix that occurred when I was formating the document

Reply:
but you are adding a resolve object?

Reply:
Yeah since that specific configuration should only exist when building the application in developer mode. So when the project is built in production, there shouldn't be any specific configuration for the development server.

Reply:
Also, this is not something that was added now, it has been there since the beginning. Just the formatting of the file caused it to move.